preview-src/docs-ndl.adoc

```diff
@@ -163,6 +177,28 @@ Tutorials & How-to guides
 xref:tutorials:index.adoc[All tutorials]
 --
 
+[.widget3.highlights]
+== GraphAcademy courses
+
+[.icon2]
+image:icons/ndl/ga-home.png[]
+
+--
+[.caption3]
+GraphAcademy courses
+
+[.list2]
+* link:{docs-home}/genai/tutorials/embeddings-vector-indexes/[Embedding & Vector Indexes Tutorial]
+* xref:{docs-home}/getting-started/tutorials/import-relational/[Import data from a relational database into Neo4j]
+* xref:{docs-home}/getting-started/tutorials/recommendation-engine/[Build a Cypher recommendation engine]
+* link:{docs-home}/operations-manual/current/tutorial/tutorial-composite-database/[Set up and use a composite database]
+* link:{docs-home}/cdc/current/procedures/query-examples/[Capture and track changes in real-time]
+* link:{docs-home}/graph-data-science-client/current/tutorials/centrality-algorithms/[Apply centrality algorithms to your graph]
+
+[.footer-link]
+xref:tutorials:index.adoc[All tutorials]
```

Comment:
I also don't understand why this link is black and the other one in the Tutorials section is blue, but both should be blue